lang/src/main/java/org/eclipse/steady/backend/BackendConnector.java

```diff
@@ -1207,7 +1222,7 @@ public void batchUpload(GoalContext _ctx) {
   		} else {
   			BackendConnector.log.error("HTTP response status [" + response.getStatus() + "], no AST for GAV [" + _lib + "] and construct " + _cid + " found");
   		}
-  	} catch (BackendConnectionException e1) {
+  	} catch (BackendConnectionException  | IOException e1) {
```

Comment:
Please also undo the changes of the two catch clauses. It is anyways in code commented out...